tests/automation/utilities/reply_message.ts

```diff
@@ -6,13 +6,26 @@ import {
   waitForTextMessage,
 } from './utils';
 
+/**
+ * Reply to a message and optionally wait for the reply to be received.
+ * @param windowSender send the message from this window
+ * @param textMessage look for this message in windowSender to reply to it
+ * @param replyText reply with this message
+ * @param windowReceiver if set, will wait until replyText is received from windowSender
```

Comment:
Same with windowReceiver

Reply:
I understand the premise i'm just not sure i agree with the naming...

Reply:
I made it even more explicit by using the object syntax

Reply:
fixed